### DISCUSSION TITLE
Please create a optional video player blocklist

### DISCUSSION THREAD

**Discussion by Bluscream** [AdGoBye/AdGoBye-Blocklists#5] [Ideas]
I never used a video player and they are always plastered around every world. So it would be great if i could just patch them out. I know at some point there was something similar to a --no-video-players command line option in vrchat but that was removed/doesnt work for ages now.

Does the blocklist format even allow for generic blocking rules? (Applies to every world where the gameobject in question is found)

Edit: From what i've seen now, it doesnt. Since the patch method needs a Content.Id which will probably be the world id in that case :/

**Comment by DubyaDude** [comment]
Blocklists are documented in the main AGB repo: https://github.com/AdGoBye/AdGoBye/blob/main/CONTRIBUTING.md
Currently, there is no form of **native** global blocking, so this is not possible.

Edit: As said below by Regalia, plugins may be your solution here.

**Reply by regalialong** [reply]
Bumping here, Plugins would do that (Plugins can declare themselves either as Global or as content specific).

I think it'd make more sense to create a Plugin for that than duplicate work on blockist entries.
